### ISSUE TITLE
Throughput and latency

### ISSUE THREAD

**Issue by H-L320** [kenzarh/SG-PBFT#1]
Hello, may I ask how to test its throughput and latency？

**Comment by kenzarh** [comment]
Hello, when you launch the main.py file, a message like this ("Client 0 got reply within 0.050984 seconds. The network exchanged 21 messages") is displayed. So we have the latency for processing one client request (0.050984 seconds). The number of clients/requests can be changed in line 9 of the main.py file. When I test it for 5 clients (line 9 becomes: requests_number = 5), it gives me the message: "Network validated 5 requests within 8.056708 seconds". And the throughput can be calculated as: 8.056708/5=1.611 seconds.

**Comment by H-L320** [comment]
OK, thank you! I have one more question, generally speaking, the latency should be on the millisecond level, but when my node increases to 100, the latency reaches seconds

**Comment by kenzarh** [comment]
Indeed, my code is not really optimized. Maybe it also depends on the environment used to launch the code. But anyway, I also get delays in seconds in an environment of about 100 nodes and more.

**Comment by naniannayu** [comment]
Hello, I would like to ask if this code was also opened in the VScode compiler or in pycharm? How can I run this project to get the experimental results in your paper：SG-PBFT: A secure and highly efficient distributed blockchain PBFT consensus algorithm for intelligent Internet of vehicles?

**Comment by naniannayu** [comment]
> Hello, may I ask how to test its throughput and latency？

I would like to ask if I need a server to run the program, or can I run it with a computer?
